### DISCUSSION TITLE
French atc et programme python

### DISCUSSION THREAD

**Discussion by chateignemoise** [Nash115/frenchATC-for-MSFS2020#32] [Ideas (and help for dev)]
Bonjour.
Je suis sous windows 11.j’ai msfs 2020.j’ai installé comme indiqué le programme python,puis le pack french atc.j’ai lancé la librairie.jusque là, pas de problèmes.je lancé mon simu,je me positionne sur un aerodrome couvert par french atc.je démarre la conversation avec les phrases prévues et , première surprise,la voix que j’entend provient de celle de mon pc et non de celle que l’on peut entendre sur vos vidéos.deuxieme surprise,l’atc me répète inlassablement la phrase:je n’ai pas compris votre demande.j’ai beau utiliser la phraseologie correcte,rien à  faire.j’ai même essayé différentes fréquences mais rien à  faire.j’utilise un  casque micro qui fonctionne bien.ma config casque est correcte . Pouvez-vous m’aider? Par avance merci. Ho ! Ho ! Ho !

**Comment by Nash115** [comment]
Bonjour !
Concernant la voix cela est normal, le programme utilise une synthèse vocal (donc celle de votre pc) a vrai dire je ne comprends pas vraiment à quelles vidéos vous faites référence :P
Sinon, concernant le fait que vous n'arrivez pas à communiquer, il n'y a pas de doute sur votre casque et micro effectivement, le problème peut venir de votre position ou de la fréquence.
Pouvez vous me donner plus d'informations ? (par exemple : votre aérodrome de départ, votre fréquence, ce que vous prononcez ....)
